kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/packages/PackagesPanelUi.java

```diff
@@ -275,6 +270,7 @@ public void onSuccess(GwtXSRFToken token) {
                             PackagesPanelUi.this.packagesFormFile.submit();
                         } else {
                             PackagesPanelUi.this.uploadModal.hide();
+                            PackagesPanelUi.this.uploadErrorText.setText(MSGS.missingFileUpload());
```

Comment:
Can you pelase change the header to: " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates"